README.md

```diff
@@ -331,6 +331,61 @@ and also by providing more type information to the Julia compiler.
   instead use `w.pymember(:member)` (for the `PyAny` conversion) or
   `w.pymember("member")` (for the raw `PyObject`).
 
+### Defining Python Classes
+
+`@pydef` creates a Python class. For instance,
+    
+    @pyimport numpy.polynomial as P
+
+    @pydef type Doubler <: P.Polynomial
+       # __init__ must return `nothing`
```

Comment:
Would be nice if the `@pydef` macro enforced this automatically.   i.e. we special-case the `__init__` function by making `__init__` point to a function whose body is `(__user's init__(args...); nothing)`.